src/shogun2-core/src/main/java/de/terrestris/shogun2/util/entity/EntityUtil.java

```diff
@@ -24,6 +24,20 @@
 public class EntityUtil {
 
 	/**
+	 * @param clazz
+	 * @param fieldName
+	 * @param forceAccess
+	 * @return
+	 */
+	public static boolean isField(Class<?> clazz, String fieldName, boolean forceAccess) {
```

Comment:
Wouldn't it be better to also check for the accepoted type, here or in another method?

Slightly related, we keep passing `forceAccess` around, but on **all**  SHOGun2 model classe we must have it `true`, since our models only have `private` fields. We should probably remove this everywhere, ansd simply pass through `true`. This could be done in another PR of course, and only if you agree.